src/SFA.DAS.RoatpGateway.Web/Settings/WebConfiguration.cs

```diff
@@ -23,6 +24,9 @@ public class WebConfiguration : IWebConfiguration
         [JsonRequired]
         public ClientApiAuthentication RoatpRegisterApiAuthentication { get; set; }
 
+        [JsonRequired]
+        public RoatpApi RoatpApi { get; set; }
```

Comment:
What is the source of this, employer config?

Reply:
yes